test/test_inspector_charm.js

```diff
@@ -210,7 +211,8 @@ describe('Inspector Charm', function() {
     assert.equal(hideSlot.calledOnce(), true);
     assert.equal(Y.one('.charmbrowser').hasClass('animate-in'), false);
     assert.equal(view.get('container').getHTML(), '');
-
+    assert.equal(destructor.callCount(), 1);
+    destructor.passThroughToOriginalMethod.call(view);
```

Comment:
There are minor test changes because the existing tests only tested the code which still remains.